lib/transform/config.js

```diff
@@ -48,10 +48,36 @@ function transformConfig(config, dictionary) {
     return dictionary.transform[name];
   });
 
+  var filesUsingTemplates = [];
+
+  var warningText = chalk.bold.red(`
+    Templates will be removed in the future, please change your config to use formats.
+    This is an example of how to fix your config.json:
+
+    Before:
+      "files": [{
+        "destination": "font_dimens.xml",
+        "template": "android/fontDimens"
+      },{
+        "destination": "colors.xml",
+        "template": "android/colors"
```

Comment:
This is a lot of text to show the user, lets just cut these examples down to 1 file.